apps/mudita-center-e2e/src/specs/help/help-window-check.e2e.ts

```diff
@@ -5,23 +5,97 @@
-      setTimeout(done, 10000)
-    })
+    const notNowButton = await HomePage.notNowButton
+    await notNowButton.waitForDisplayed()
+    await notNowButton.click()
   })
-  it("Clicks help tab, switches to new window and checks if help topic is displayed in new window", async () => {
+
+  it("Open Help window", async () => {
     const helpTab = await NavigationTabs.helpTab
     await helpTab.waitForDisplayed({ timeout: 15000 })
     await helpTab.click()
 
     await browser.switchWindow("#/help")
+  })
+
+  it("Check contents of Mudita Help", async () => {
+    // Check window title
+    const helpTitle = await HelpPage.windowTitle
+    await helpTitle.waitForDisplayed({ timeout: 15000 })
+    await expect(helpTitle).toBeDisplayed()
+    await expect(helpTitle).toHaveText("Mudita Center Help")
+
+    // Check presence of the search engine
+    const searchIcon = await HelpPage.searchIcon
+    await searchIcon.waitForDisplayed()
+    await expect(searchIcon).toBeDisplayed()
+
+    const searchPlaceholder = await HelpPage.searchPlaceholder
+    await expect(searchPlaceholder).toHaveAttributeContaining("placeholder", "Search")
+
+    // Check presence of Contact support button
+    const contactSupportButton = await HelpPage.contactSupportButton
+    await contactSupportButton.waitForDisplayed()
+    await expect(contactSupportButton).toBeDisplayed()
+    await expect(contactSupportButton).toBeClickable()
+
+    const contactSupportTooltip = await HelpPage.contactSupportButtonTooltip
+    contactSupportButton.moveTo()
+    await expect(contactSupportTooltip).toBeDisplayed()
+    await expect(contactSupportTooltip).toHaveText("Contact support")
 
-    const helpTopic = await HelpPage.helpListElement
-    await helpTopic.waitForDisplayed({ timeout: 15000 })
+    // Check accordion
+    const helpTopic = await HelpPage.listElement
+    await helpTopic.waitForDisplayed()
     await expect(helpTopic).toBeDisplayed()
+    const no_of_articles = await $$('[data-testid="help-component-question"]').length
+    await expect(no_of_articles).toEqual(25)
+  })
+
+  it("Check content of first article", async () => {
+    const helpTopic = await HelpPage.listElement
+    await expect(helpTopic).toHaveText("How to import my iCloud contacts into Mudita Pure by using .vcf file?")
+    await helpTopic.click()
+    const helpTopicContent = await HelpPage.topicContent
+    await expect(helpTopicContent).toBeDisplayed()
+    await expect(helpTopicContent).toHaveTextContaining('Click on the “Import” button.')
+    const backLink = await HelpPage.articleBackLink
+    await backLink.click()
+    const helpTitle = await HelpPage.windowTitle
+    await expect(helpTitle).toHaveText("Mudita Center Help")
+  })
+
+  it("Search for questions & check search results", async () => {
+    const searchPlaceholder = await HelpPage.searchPlaceholder
+    searchPlaceholder.setValue("fail")
+    browser.keys("\uE007")
+    const helpTopic = await HelpPage.listElement
```

Comment:
There is an error from Typescript in line`82`
`Cannot redeclare block-scoped variable 'helpTopic'.`
You are declaring helpTopic twice. 

```suggestion
let helpTopic = await HelpPage.listElement
```

Reply:
I think that it's cleaner to use const and rename the second 'helpTopic' to 'filteredHelpTopic'. Additionally name will be more meaningful.

Reply:
Okay